libraries/chain/include/eosio/chain/chain_snapshot.hpp

```diff
@@ -16,10 +16,9 @@ struct chain_snapshot_header {
     *         - WebAuthn keys
     *         - wtmsig block siganatures: the block header state changed to include producer authorities and additional signatures
     *         - removed genesis_state and added chain ID to global_property_object
-    *   4: Updated for v3.0.0 protocol features:
+    *   4: Updated for v2.1.0 protocol features:
     *         - forwards compatible with versions 2 and 3
```

Comment:
Should 3 be changed to 2.1.0?

Reply:
On line 20? The 3 there is referencing chain_snapshot_header version 3, which is independent of nodeos version.